### ISSUE TITLE
[Scratch Messaging] Requests blocked by CORS

### ISSUE THREAD

**Issue by micahlt** [ScratchAddons/ScratchAddons#910]
**Describe the bug**

When using ScratchAddons Scratch Messaging, the background page's logs show nothing but a whole lot of blocked requests.  It seems to be an issue with CORS policy.  

**To Reproduce**

1. Install the extension
2. Try to use Scratch Messaging

**Expected behavior**

Messages appear in the extension's popup.

**Screenshots**

![image](https://user-images.githubusercontent.com/28816297/100884140-bf960f80-3476-11eb-9d15-8b0b6290b15c.png)

**Device (please complete the following information):**

- OS: Windows 10
- Browser: Chrome
- Version: 86

**Additional context**

I've had similar issues with CORS policy on the Scratch API.  It may be easily solved by integrating a [CORS proxy](https://github.com/Rob--W/cors-anywhere).

**Comment by TheColaber** [comment]
I think I know you situation here:
1. Right click the scratch addons icon in the extensions bar
2. Click "manage extensions"
3. Make sure these all four are checked:
![](https://user-images.githubusercontent.com/17484114/99267611-eca1bb80-2802-11eb-97ca-fd95fb8cd2db.png)

**Comment by TheColaber** [comment]
Try above @micahlt if those checks aren't checked please let me know.

**Comment by micahlt** [comment]
Checking all of them does change the situation.  However, now I see the reload button, and clicking it does nothing: 

![image](https://user-images.githubusercontent.com/28816297/100885545-63cc8600-3478-11eb-9d76-ec39576d1cde.png)

I solved this by open the ScratchAddons log and running `window.location.reload()`.  Everything works fine!  Thanks for your help @TheColaber!

**Comment by TheColaber** [comment]
No prob. We will have a prompt for these steps later on our website just because this has happened before. Thanks for posting an issue on the repo!

**Comment by micahlt** [comment]
What probably needs to happen is having an error message to handle CORS errors that tells the user to turn on those settings.  Perhaps I'll make a PR 😄

**Comment by WorldLanguages** [comment]
@TheColaber @micahlt I'd suggest you first PR in the website repo, so that we can open a page like scratchaddons.com/broken (or something like that) in a new tab as a way to tell the user SA isn't working and how to fix it.

**Comment by WorldLanguages** [comment]
Duplicate of #784
